ledger-core/virtual/object/object.go

```diff
@@ -398,14 +404,74 @@ func (sm *SMObject) createWaitPendingOrderedSM(ctx smachine.ExecutionContext) {
 	}
 }
 
-func (sm *SMObject) migrate(ctx smachine.MigrationContext) smachine.StateUpdate {
+func (sm *SMObject) stepMigration(ctx smachine.MigrationContext) smachine.StateUpdate {
 	if sm.GetState() == Unknown {
 		ctx.Log().Trace("SMObject migration happened when object is not ready yet")
 		return ctx.Stop()
 	}
 
 	ctx.UnpublishAll()
 
+	// store migration pulse
+	sm.MigratePulseNumber = sm.pulseSlot.PulseData().PulseNumber
+
+	sdl := ctx.Share(&sm.SummaryDone, 0)
+
+	if !ctx.Publish(SummarySyncKey{
+		objectRef:   sm.Reference,
+		pulseNumber: sm.MigratePulseNumber,
```

Comment:
pulseNumber could be omited. This syncKey lives in one pulse slot. There should not be situation when we'll have two keys for one object in one pulseSlot

Reply:
Fixed